addons/pinniped/tanzu-auth-controller/main.go

```diff
@@ -24,7 +24,7 @@ func main() {
 	klog.InitFlags(nil)
 	flag.Parse()
 	ctrl.SetLogger(klogr.New())
-	setupLog := ctrl.Log.WithName("pinniped config controller").WithName("set up")
+	setupLog := ctrl.Log.WithName("tanzu auth controller").WithName("set up")
```

Comment:
Should this be `tanzu auth controller manager`, because it manages the various tanzu auth controllers?

Reply:
yup! I'll change it.